packages/plugin-board/README.md

```diff
@@ -0,0 +1,192 @@
+# addContent
```

Comment:
Delete this and replace it with a short description. READMEs are no long autogenerated, but they'll get expanded by hand over time.